src/renderers/native/ReactNativeFiberRenderer.js

```diff
@@ -377,6 +377,11 @@ const NativeRenderer = ReactFiberReconciler({
   },
 
   useSyncScheduling: true,
+
+  now(): number {
```

Comment:
Why is this added to `ReactNativeFiberRenderer` and not `ReactNativeFiberEntry`?

Reply:
"Entry" modules are the public API. The renderer is what you create out of a host config. For whatever reason, our React Native renderer splits the two into separate files, whereas our DOM renderer does not. Don't think there's any real reason, just happened to be factored that way.